### PR TITLE
Completed rough general layout

### PR DESCRIPTION
**Pull request by luke-grayland**
I've completed the general layout (nav bar, header, about me, projects, footer) and will be adding content and further detail next. The lorem ipsum text can be ignored for now. Thanks!

### CODE REVIEW

File: portfolioMain.html [hunk 1]
```diff
@@ -0,0 +1,72 @@
+<!DOCTYPE html>
+<html>
```
Comment (j-add) [inline portfolioMain.html:2]: Could do with a `lang` attribute

**Review by j-add — CHANGES_REQUESTED** [review]
Looking good, just needs some small changes and content adding to footer

File: portfolioMain.html [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+<head>
```
[the remark this answers is not in the document]
Comment (luke-grayland) [reply portfolioMain.html:3]: Good point, have added one

File: portfolioMain.html [hunk 3]
```diff
@@ -0,0 +1,72 @@
...
+			-->
+		</div>
+		<div class="dividerDiv">
+		</div>
+		
+	</main>
+</body>
+<footer>
+	<div class="footer">
+		<p>I am a footer </p>
```
[the remark this answers is not in the document]
Comment (luke-grayland) [reply portfolioMain.html:68]: Yes, I'll add some content now

File: portfolioStyle.css [hunk 4]
```diff
@@ -0,0 +1,159 @@
...
+	text-decoration: none;
+	
+}
+
+.navList :visited {
+	color: var(--darkBlue);
+}
+
+.navList :hover {
+	color: var(--greyBlue);
```
[the remark this answers is not in the document]
Comment (luke-grayland) [reply portfolioStyle.css:67]: Good shout - I'll change it from switching to the lighter colour

File: portfolioStyle.css [hunk 5]
```diff
@@ -0,0 +1,159 @@
...
+}
+
+.aboutMeImage {
+	height: auto;
+	width: 500px;
+
+}
+
+.aboutMeImageDiv {
+	margin: auto auto;
```
[the remark this answers is not in the document]
Comment (luke-grayland) [reply portfolioStyle.css:119]: Ah cool, I didn't know that

**Review by gabrielrowan — COMMENTED** [review]
I could only find 1 thing! A double semi colon - but not sure if this is intentional for that particular style type

**Pull request merged**